support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -142,12 +140,6 @@ const paymentFrequencyButtonsCss = css`
 	}
 `;
 
-const tabletLineBreak = css`
```

Comment:
This does mean we lose the line-break in the heading below desktop. We could solve that in the tooling, but we should also reconsider how important it is